README.md

```diff
@@ -1,15 +1,23 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103-W17-1/tp/actions)
```

Comment:
URL needs to change from `https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg` to `https://github.com/AY2122S2-CS2103-W17-1/tp/workflows/Java%20CI/badge.svg`

Reply:
thanks for catching, resolved in upcoming commit